austinChappell/nhl-react
# Tech/add linter

## Overview

As a developer, I want to have consistent style between a team of developers.

## Solution

Add eslint and use husky. Run the linter on commits and tests on pushes.

## To test

- Get this code
- Run `yarn`
- Run `yarn eslint`
- Linting should pass
- Run `yarn test`
- All tests should pass
